tests/unit/src/test/scala/tests/worksheets/WorksheetSuite.scala

```diff
@@ -125,8 +122,7 @@ class WorksheetSuite extends FunSuite with BeforeAndAfterAll with DiffAssertions
       |Stream.from(10)
       |""".stripMargin,
     """|
-       |<Stream.from(10)> // Stream(10,11,12,...
-       |
+       |<Stream.from(10)> // Stream(10,11,12,13,1...
```

Comment:
@olafurpg I was not sure, if summary supposed to be shorten in case, when details are shorter because of the height. Can fix it.

Reply:
Previously we included the “ // “ header in the width of the message.

Reply:
It’s fine to keep this PR unchanged here, clients can pass in a smaller width if they want extra horizontal space for a header

Reply:
I meant that details are only
```
Stream(
       |  10,
       |  11,
       |  12,
       |...
``` -- less information, because of height constraint -- I removed that constraint for the summary, since it's one-line and is bounded by the width.